v19.2/collate.md

```diff
@@ -128,6 +129,47 @@ You can also use casting to remove collations from values.
 ~~~ sql
 > SELECT CAST(name AS STRING) FROM de_names ORDER BY name;
 ~~~
+~~~
+    name
++----------+
+  Backhaus
+  Baz
+  Bär
+(3 rows)
+~~~
+
+### Show collation for strings
+
+<span class="version-tag">New in v19.2:</span> You can use the `pg_collation_for` [built-in function](functions-and-operators.html#string-and-byte-functions), or its alternative [syntax form](functions-and-operators.html#special-syntax-forms) `COLLATION FOR`, to return the collation of a collated string.
```

Comment:
The couple things:
1. I think this should say "locale name of a collated string".  According to the logic test I'm seeing at https://github.com/cockroachdb/cockroach/pull/39198/files this function returns the "locale name of a collated string".  However here we're saying the "collation of a collated string". Is that the same thing?  In any case, if I don't know what collation is it's a bit of a circular definition.
2. Re: the new syntax form, pretty sure the SQL grammar needs to be updated also.  Feel free to ping me if you need help with that.  I haven't done it in quite a while so we can learn/remember together.